src/content/structured/components/badge/guidance.mdx

```diff
@@ -0,0 +1,243 @@
+---
+path: "/components/badge"
+
+navPriority: 3
+
+# date: "2023-02-03"
```

Comment:
This date should be uncommented, will need updating to a more recent date too

Reply:
Good spot!